### PR TITLE
Axosoft.GitKraken version 8.3.0

### PR DESCRIPTION
**Pull request by Daninator1**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/49256)

### CODE REVIEW

File: manifests/a/Axosoft/GitKraken/8.3.0/Axosoft.GitKraken.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- silent
+Installers:
+- InstallerLocale: en-US
+  Platform:
+  - Windows.Desktop
+  Architecture: x64
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://release.axocdn.com/win64/GitKrakenSetup-8.3.0.exe
+  InstallerSha256: 5B4AD08FF40B76301D54DFC94F28E82442EB322748B3DEFB71BDA5C04C2BC1B1
```
Comment (ItzLevvie) [inline manifests/a/Axosoft/GitKraken/8.3.0/Axosoft.GitKraken.installer.yaml:18]:
```suggestion
  InstallerSha256: 1A12929226EBB49FAC0BCE1712EECC2AC7735FF03CEFB512147F38F747000892
```

File: manifests/a/Axosoft/GitKraken/8.3.0/Axosoft.GitKraken.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    SilentWithProgress: /silent
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Platform:
+  - Windows.Desktop
+  Architecture: x86
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://release.axocdn.com/win32/GitKrakenSetup-8.3.0.exe
+  InstallerSha256: 5B4AD08FF40B76301D54DFC94F28E82442EB322748B3DEFB71BDA5C04C2BC1B1
```
Comment (ItzLevvie) [inline manifests/a/Axosoft/GitKraken/8.3.0/Axosoft.GitKraken.installer.yaml:30]:
```suggestion
  InstallerSha256: 1A12929226EBB49FAC0BCE1712EECC2AC7735FF03CEFB512147F38F747000892
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/a/Axosoft/GitKraken/8.3.0/Axosoft.GitKraken.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Axosoft.GitKraken
+PackageVersion: 8.3.0
+PackageLocale: en-US
+Publisher: Axosoft, LLC
```
Comment (ItzLevvie) [inline manifests/a/Axosoft/GitKraken/8.3.0/Axosoft.GitKraken.locale.en-US.yaml:7]:
```suggestion
Publisher: GitKraken
```
This must match Control Panel.

Comment (ItzLevvie) [reply manifests/a/Axosoft/GitKraken/8.3.0/Axosoft.GitKraken.locale.en-US.yaml:7]: image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by Daninator1** [comment]
How can i validate the download links? I just adapted the previous urls which point to the company's own CDN where they host the software.

**Comment by denelon** [comment]
@Daninator1 can you provide a link to the publisher's website referencing the URL in the manifest? If we can confirm the URL is the official one from the publisher, we can add a waiver for this to pass.

**Comment by Daninator1** [comment]
Problem is that the official link on the website (https://www.gitkraken.com/download/windows64) is this one https://release.gitkraken.com/win64/GitKrakenSetup.exe which does not contain the version number.
However, appending the version (e.g. 8.3.0) to the setup .exe does work and provides the specific version, so maybe this would also be a valid url to use?

**Comment by msftrubengu** [comment]
We already have a version using that domain, which was validated last December #37640

**Pull request merged**